src/page/Profile/index.tsx

```diff
@@ -84,12 +94,26 @@ const ProfilePage = () => {
                       />
                     </FormGroup>
                   </Form>
+                  <UncontrolledAlert 
+                    className="mb-0" color="success" 
+                    isOpen={profileUpdated} 
+                    toggle={() => setProfileUpdated(false)}
+                  >
+                    Cập nhật thành công
+                  </UncontrolledAlert>
```

Comment:
- Use `profileUpdating` for `isOpen` props, no need another state `profileUpdated`.
- Need to define an `error` state (contain form error). Use `error` value to render `color` props of `<UncontrolledAlert />`